.cursor/skills/dev-helper/SETUP.md

```
@@ -0,0 +1,206 @@
+gh repo view "$GH_REPO" --json name -q .name
+
+# 5. Hooks are executable
+.cursor/hooks/dev-helper-state.sh
+.cursor/hooks/personal-shell-allowlist.sh <<< '{"command": "echo test"}'
+```
+
+---
+
+## Directory Structure After Setup
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)